src/main/resources/jenkins/plugins/office365connector/WebhookJobProperty/config.jelly

```diff
@@ -1,100 +1,9 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-<!--
-   	 This jelly script is used for per-project configuration.
-
-	See global.jelly for a general discussion about jelly script.
--->
-
-<!--
-	Creates a text field that shows the value of the "name" property.
-	When submitted, it will be passed to the corresponding constructor parameter.
--->
   	<f:section title="Office 365 Connector">
   		<f:entry title="Notification webhooks" field="webhooks">
-  			<f:repeatable name="webhooks" var="webhook"
-  				items="${instance.getWebhooks()}" add="${%Add Webhook}">
-  				<table>
-					<f:entry field="webhook">
-
-						<f:entry title="Name" field="name">
-							<f:textbox name="name" value="${webhook.getName()}" />
```

Comment:
Tedious use of name and value... when we can have Webhook described and use fields 👍